hydra-redis/querying_mechanism.py

```diff
@@ -0,0 +1,184 @@
+import redis
+import hydra_graph as graph
+from hydrus.hydraspec import doc_maker
+import hydrus
+from collections_endpoint import CollectionEndpoints
+import urllib.request
+import json
+
+def final_file(url):
+    """ Open the given url and read and load the Json data."""
+    response = urllib.request.urlopen(url)
+    return json.loads(response.read().decode('utf-8'))
+
+def split_data(count,objects):
+    for obj in objects:
+        string = obj.decode('utf-8')
+        map_string = map(str.strip,string.split(','))
+        property_list = list(map_string)
+        check = property_list.pop()
+        property_list.append(check.replace("\x00",""))
+        print (property_list)
+
+def endpoints_query(query):
+    r = redis.StrictRedis(host='localhost', port=6379, db=0)
```

Comment:
You should factorized this at application run-time, and make the connection available through a proxy class or function like:
```
class RedisProxy:
    def __init__(self):
        self.connection = redis.StrictRedis...
    def get_connection(self):
        return self.connectoin
```